src/Module.php

```diff
@@ -275,7 +275,7 @@ public function getReplyToAttribute()
             } else {
                 // try to auto detected email attribute from attributes last
                 foreach (['email', 'mail', 'emailaddresse', 'email_address'] as $mail) {
-                    if (in_array($mail, $this->attributes)) {
+                    if (isset($this->attributes) && in_array($mail, $this->attributes)) {
```

Comment:
maybe we should take `empty` instead of `isset`?

Reply:
or we could set attributes to an empty array by default? `public $attributes = []`.